docs/home/whats-new-2-17.md

```diff
@@ -0,0 +1,145 @@
+
+#### Authentication and Authorization
+
+* <!-- RUN-15585/RUN-15586 -  View & Edit SSO settings - SAML -->Added new functionality to SAML 2 in the *SSO configuration* in the *Security* category* of the *Tools & Settings* menu. The added functionality assists with troubleshooting login or permissions issues that may arise. Now administrators now have the ability to:
+
+    * View and edit SSO settings.
+    * Upload or download a settings XML file.
+
+For more information, see [SSO UI configuration](../admin/runai-setup/authentication/sso.md#step-1-ui-configuration).
+
+<!-- #### Policies - 
+no new stories here
+
+### Control and Visibility
+
+no new stories here -->
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform.
+
+### Feature deprecations
+
+Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative. The following features have been marked for deprecation:
+
+### API support and endpoint deprecations
+
+The endpoints and parameters specified in the API reference are the ones that are officially supported by Run:ai. For more information about Run:ai's API support policy, see [Developer overview](../developer/overview-developer.md#overview-developer-documentation).
+
+The following list of API endpoints have been marked for deprecation:
+
+* Jobs&mdash;the *Jobs* feature (submission form and view) has been moved to the category of *Legacy*. To enable them, go to *Tools & Settings*, *General*, open the *Workloads* pane, and then toggle the *Jobs view* and *Job submission* switch to the enabled position.
+* Deployments&mdash;the *Deployments* feature has been removed. It has been replaced by *Inference* workloads. For more information, see [Jobs, Workloads, and Workspaces](#jobs-workloads-and-workspaces) above.
+* Workspaces view&mdash;the *Workspaces* menu has been removed. You can now submit a *Workspace* workload using the *+ New workload* form from the *Workloads* table.
+
+#### Jobs, events, pods API (replaced by workloads/pods/events)
+
+| Deprecated endpoint | Replacement endpoint |
+| -- | -- |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs | https://app.run.ai/api/v1/workloads |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs/count | https://app.run.ai/api/v1/workloads/count |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/jobs/{jobId}/pods | https://app.run.ai/api/v1/workloads/{workloadId}/pods |
+| https://app.run.ai/v1/k8s/clusters/{uuid}/pods | https://app.run.ai/api/v1/workloads/pods | 
+
+#### cluster metrics (v1) - replaced v2
+
+| Deprecated endpoint | Replacement endpoint |
+| -- | -- |
+| https://app.run.ai/v1/k8s/clusters/{clusterUuid}/metrics | https://app.run.ai/api/v2/clusters/{clusterUuid}/metrics |
+
+In some cases name of metrics have been changed. For more information, see [Metrics](../developer/metrics/metrics.md#changed-metrics-and-api-mapping).
```

Comment:
Should be in a section of it's own. It's important that metrics users will understand that starting cluster version 2.17 some names have changed. It's a breakage they need to be aware of.

Reply:
@jasonnovichRunAI